x/crosschain/module.go

```diff
@@ -154,6 +155,9 @@ func (am AppModule) RegisterServices(cfg module.Configurator) {
 	if err := cfg.RegisterMigration(types.ModuleName, 4, m.Migrate4to5); err != nil {
 		panic(err)
 	}
+	if err := cfg.RegisterMigration(types.ModuleName, 13, m.Migrate13to14); err != nil {
```

Comment:
Use actual values here instead of something like `common.AppVersion-1` so that we avoid issues around running the wrong handler